src/main/java/ch/njol/skript/expressions/ExprLastLoginTime.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.OfflinePlayer;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Date;
+import ch.njol.util.Kleenean;
+
+@Name("Last/First Login Time")
+@Description("When a player last/first logged in the server.")
+@Examples({"command /onlinefor:",
+	"\ttrigger:",
+	"\t\tsend \"You have been online for %difference between player's last login date and now%.\"",
+	"\t\tsend \"You first joined the server %difference between player's first login date and now% ago.\""})
+@Since("INSERT VERSION")
```

Comment:
The last login method requires Paper 1.13 so it may be good to mention it with the RequiredPlugins annotation. Or at least mentioning the behaviour change in the description.

Reply:
I'm not sure as to what getLastPlayed does, as in whether or not it gets when they last joined or when they last left the server

Reply:
In Paper, it was replaced by OfflinePlayer#getLastLogin and OfflinePlayer#getLastSeen, since the behaviour of the method is unclear, my wild guess is that it is the time they were last seen.